src/config.c

```diff
@@ -3071,7 +3071,7 @@ standardConfig static_configs[] = {
     createBoolConfig("lazyfree-lazy-expire", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG, server.lazyfree_lazy_expire, 0, NULL, NULL),
     createBoolConfig("lazyfree-lazy-server-del", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG, server.lazyfree_lazy_server_del, 0, NULL, NULL),
```

Comment:
I guess I wonder if we should change these as well? What is the motivation for keeping these as synchronous?

Reply:
I think we can enable all lazyfree configs.